src/main/scala/rocket/ICache.scala

```diff
@@ -23,6 +23,8 @@ case class ICacheParams(
     nWays: Int = 4,
     rowBits: Int = 128,
     nTLBEntries: Int = 32,
+    nTLB4KSectors: Int = 4,
```

Comment:
I'd consider calling this `nTLBBasePageSectors` (more symmetric with "Superpage" and a more generic name overall).  LGTM otherwise

Reply:
Along those lines, the preexisting nTLBEntries should really be nTLBBasePageEntries, but that's too intrusive of a change for now I would hazard....

Reply:
Yeah, agreed on both counts.